### PR TITLE
5/21 Release notes

### PR DESCRIPTION
**Pull request by lzdanski**
Related PR: https://github.com/astronomer/docs/pull/3705

### CODE REVIEW

**Review by aliotta — APPROVED** [review]

File: astro/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+The ability to deploy automatically from GitHub using the official Astro GitHub integration is now in Public Preview.
+
+The Astro GitHub integration is a new way to automatically deploy code from a GitHub repository to Astro by merging pull requests or making commits directly to specific branches, without needing to configure a GitHub Action. Additionally, the GitHub integration displays Git metadata directly in the Astro UI, including Git commit descriptions and gives you greater visibility into the status and logs of individual code deploys.
+
+See [Deploy code with the Astro GitHub integration](deploy-github-integration.md) for more information.
+
+### Bug fixes
+
+- Fixed a bug where users creating custom roles in the UI could see roles that could not be assigned. <!-- https://github.com/astronomer/astro/pull/21148-->
```
Comment (aliotta) [inline astro/release-notes.md:43]: "users creating custom roles" should say "users using custom roles"

Comment (lzdanski) [inline astro/release-notes.md:43]:
```suggestion
- Fixed a bug where users using custom roles in the UI could see roles that could not be assigned.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -25,6 +25,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## May 21, 2024
+
+### Official Astro GitHub integration is now in Public Preview
```
Comment (jwitz) [inline astro/release-notes.md:30]:
```suggestion
### The Astro GitHub integration is now in Public Preview
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -25,6 +25,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+The ability to deploy automatically from GitHub using the official Astro GitHub integration is now in Public Preview.
+
+The Astro GitHub integration is a new way to automatically deploy code from a GitHub repository to Astro by merging pull requests or making commits directly to specific branches, without needing to configure a GitHub Action. Additionally, the GitHub integration displays Git metadata directly in the Astro UI, including Git commit descriptions and gives you greater visibility into the status and logs of individual code deploys.
+
+See [Deploy code with the Astro GitHub integration](deploy-github-integration.md) for more information.
+
+### Bug fixes
+
+- Fixed a bug where users using custom roles in the UI could see roles that could not be assigned.
```
Comment (jwitz) [inline astro/release-notes.md:43]:
```suggestion
- Fixed an issue where users with custom roles could see roles that could not be assigned in the Astro UI.
```
Where are they seeing the roles? Is this users that have custom roles, or users who are creating/ assigning custom roles? Some clarifying details here would be helpful

**Review by jwitz — APPROVED** [review]

**Comment by jwitz** [comment]
We also have to  update updates.astronomer.io!

File: astro/release-notes.md (hunk 3 above)
Comment (lzdanski) [reply astro/release-notes.md:43]: The clarifying details you added are my understanding of the bug fix!

File: astro/release-notes.md [hunk 4]
```diff
@@ -25,6 +25,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## May 21, 2024
```
Comment (lzdanski) [inline astro/release-notes.md:28]:
```suggestion
## May 22, 2024
```

**Pull request merged**